_sass/blocks/case-studies/_intro.scss

```diff
@@ -41,3 +41,19 @@
     height: 40px;
   }
 }
+
+.case_studies-maps {
+  path.mesh { //county outlines
+    color: $neutral-gray;
+  }
+  path.feature { //highlighted county features
+    color: $green;
+  }
+}
+
+.case_studies-selected_counties {
+  path { //highlighted county features
+    fill: $green;
+    stroke: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.